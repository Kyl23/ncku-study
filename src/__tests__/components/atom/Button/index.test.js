import { render } from '@testing-library/react';
import Button from '~/src/components/atom/Button';

describe('test for atom - Button', () => {
    it('render Badge default theme must be same', () => {
        const { container } = render(<Button theme="primary" />);

        expect(container).toMatchSnapshot();
    });

    it('render Badge with specify props must be same', () => {
        const props = {
            height: '20px',
            weight: '100px',
            theme: 'light',
        };

        const { container } = render(<Button {...props} />);
        expect(container).toMatchSnapshot();
    });
});
