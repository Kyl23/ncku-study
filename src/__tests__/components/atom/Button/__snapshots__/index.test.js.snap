// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for atom - Button render Badge default theme must be same 1`] = `
<div>
  <button
    class="css-1tlwuer"
  />
</div>
`;

exports[`test for atom - Button render Badge with specify props must be same 1`] = `
<div>
  <button
    class="css-jb9n5v"
    height="20px"
  />
</div>
`;
